pyproject.toml

```
@@ -28,6 +28,7 @@ dependencies = [
   "cryptography >= 42",
   "id >= 1.1.0",
   "importlib_resources ~= 5.7; python_version < '3.11'",
+  "pyasn1",
```

Nitpick: we probably don't need a strong constraint here, but this might dodge weirdness where someone tries to drop `sigstore` into a pre-existing env:

```suggestion
  "pyasn1 ~= 0.6",
```